src/cmd/manage_nsfs.js

```diff
@@ -431,24 +541,36 @@ async function manage_account_operations(action, data, config_root, config_root_
 
 
 /**
- * list_config_file will list all the config files (json) in a given path
- * @param {fs.PathLike} full_path
+ * list_config_file will list all the config files (json) in a given config directory
+ * @param {string} config_path
```

Comment:
Why do we change it to string?
Isn't it always a path?

Reply:
this is the signature of join(), - join(...paths: string[]): string;